src/Utils/Converter.php

```diff
@@ -72,7 +72,7 @@ class Converter implements LoggerAwareInterface {
     'address' => TRUE,
     'applet' => TRUE,
     'area' => TRUE,
-    'audio' => TRUE,
+    // 'audio' => TRUE,
```

Comment:
If I'm very picky here, the MDN docs don't suggest that `audio` is block level.
https://developer.mozilla.org/en-US/docs/Web/HTML/Block-level_elements